test/jdk/tools/jmod/hashes/HashesTest.java

```diff
@@ -81,17 +74,20 @@
             new RuntimeException("jar tool not found")
         );
 
-    private final Path mods;
-    private final Path srcDir;
-    private final Path lib;
-    private final ModuleInfoMaker builder;
+    static final AtomicInteger counter = new AtomicInteger(0);
+
+    private Path mods;
+    private Path lib;
+    private ModuleInfoMaker builder;
 
-    HashesTest(Path dest) throws IOException {
+    @BeforeMethod
+    public void setTestPath() throws IOException {
+        Path dest = Path.of("test" + counter.addAndGet(1));
         if (Files.exists(dest)) {
             deleteDirectory(dest);
         }
         this.mods = dest.resolve("mods");
-        this.srcDir = dest.resolve("src");
+        Path srcDir = dest.resolve("src");
```

Comment:
you can just get rid of this local variable and do this in line 92:
```
     this.builder = new ModuleInfoMaker(dest.resolve("src"));
```

Reply:
Yes I can do that and realized that after I pushed the update